projects/cm_mcu/Tasks.h

```diff
@@ -233,6 +233,9 @@ int SystemStackWaterHighWaterMark();
 // clock IO expander initalization
 void init_registers_clk();
 
+// configuring clock initalization
+int init_load_clk(int clk_n);
```

Comment:
I had understood that we were going to have the interface allow for different programs for each clock...

Reply:
We will! but let me make the triplet version works first since the way we organize EEPROM now is only for one program stored.

Reply:
It might make sense to set up the right interface from the start....

Reply:
we have decided to store the different config files on Zynq instead because EEPROM will programed via Zynq anyway.